packages/js/components/src/experimental-tree-control/types.ts

```diff
@@ -1,7 +1,7 @@
 export interface Item {
 	parent?: string;
-	value: string;
-	label: string;
+	id: string;
```

Comment:
Instead of changing these props why you didn't do generic here. Like in the `select-control`, `select-tree-control` and `selected-items` components, at the end you make use of the following function anyway because of that right. 
```typescript
getItemLabel?: getItemLabelType< Item >;
getItemValue?: getItemValueType< Item >;
```
The problem with generic types is that you need to let the component knows how to get access to the `label` and `value` props within the passed object right. But you already have that! So we can apply the same to the Tree so we can pass a generic type instead of fixed type (`Item`).
The advantage of this is that we can pass any object list down to the tree.

Reply:
I decided to rename it back and not use generics for now. You're correct that it didn't make sense having getItemLabel and getItemValue as props on the component the way it previously was, so I removed it. It would be a considerate effort to implement generics in the Tree control because the `Item` type is heavily used throughout the component, for example in `useLinkedTree` and `useSelection`